cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+COPY ./examples/hotrod/hotrod-demo-linux /bin/
+COPY ./cmd/standalone/standalone-linux /bin/
+
+ENTRYPOINT /bin/hotrod-demo-linux all & /bin/standalone-linux --span-storage.type=memory --query.port=3001
+
+#example build command from github.com/uber/jaeger directory: docker build -f cmd/standalone/Dockerfile -t standalone .
```

Comment:
right now, docker_build command depends on the user having jaeger-ui and npm, etc. For now I'm not going to expose it in the makefile. Will do in the future when we make jaeger-ui a submodule.